src/components/AccountCreation.jsx

```diff
@@ -0,0 +1,185 @@
+import { useState, useRef } from 'react';
+import { Form, Button, Card, Alert } from 'react-bootstrap';
+import FilterDropdown from './common/FilterDropdown';
+
+function AccountCreation() {
+  // @TODO Fetch the schools from the database
+  // For now mock the data
+  const schools = [
+    'Tufts University',
+    'Ohio State University',
+    'Tower Hill School',
+    'Wesleyan University'
+  ];
+  const [school, setSchool] = useState(schools[0]);
```

Comment:
Nit: I feel we should make this not default to a school and handle a null state to make sure is intentional about what school they select, but its not a big deal